stock_barcodes/wizard/stock_barcodes_read_picking.py

```diff
@@ -57,6 +59,23 @@ def _compute_todo_line_display_ids(self):
         """
         self.todo_line_display_ids = self.todo_line_id
 
+    @api.depends(
+        "picking_id", "barcode", "picking_id.move_lines.move_line_ids.qty_done"
+    )
+    def _compute_pending_move(self):
+        for record in self:
+            text = ""
+            if record.picking_id:
+                moves = record.picking_id.move_ids_without_package.filtered(
+                    lambda r: r.product_uom_qty > r.quantity_done
+                )
+
+                text = self.env["ir.qweb"].render(
+                    "stock_barcodes.missing_moves",
+                    {"picking": record.picking_id, "moves": moves},
+                )
+            record.pending_moves = text
+
```

Comment:
In v12.0 @albariera added the config to do that optional.
In this method you must check the security group due to performance